glide.yaml

```diff
@@ -25,6 +25,8 @@ import:
 - package: github.com/go-stack/stack
   version: ~1.5.2
 - package: github.com/coreos/etcd
-  version: ~3.1.0-rc.1
+  version: 3.2.1
   subpackages:
   - client
+- package: golang.org/x/text
+  version: 4ee4af566555f5fbe026368b75596286a312663a
```

Comment:
Here I updated etcd. The text dependency had to be pinned because there was some version conflict.